dwave/system/composites/virtual_graph.py

```diff
@@ -164,32 +166,35 @@ def sample(self, bqm, apply_flux_bias_offsets=True, **kwargs):
                 Optional keyword arguments for the sampling method, specified per solver.
 
         Examples:
-           This example uses :class:`.VirtualGraphComposite` to instantiate a composed sampler
-           that submits an Ising problem to a D-Wave solver.
-           The problem represents a logical
-           NOT gate using penalty function :math:`P = xy`, where variable x is the gate's input
-           and y the output. This simple two-variable problem is manually minor-embedded
-           to a single :std:doc:`Chimera <oceandocs:docs_system/intro>` unit cell: each variable
-           is represented by a chain of half the cell's qubits, x as qubits 0, 1, 4, 5,
-           and y as qubits 2, 3, 6, 7.
-           The chain strength is set to half the maximum allowed found from querying the solver's extended
-           J range. In this example, the ten returned samples all represent valid states of
-           the NOT gate.
-
-           >>> from dwave.system import DWaveSampler, VirtualGraphComposite
-           >>> embedding = {'x': {0, 4, 1, 5}, 'y': {2, 6, 3, 7}}
-           >>> qpu_2000q = DWaveSampler(solver={'topology__type': 'chimera'})
-           >>> qpu_2000q.properties['extended_j_range']
-           [-2.0, 1.0]
-           >>> sampler = VirtualGraphComposite(qpu_2000q, embedding, chain_strength=1) # doctest: +SKIP
-           >>> h = {}
-           >>> J = {('x', 'y'): 1}
-           >>> sampleset = sampler.sample_ising(h, J, num_reads=10) # doctest: +SKIP
-           >>> print(sampleset)    # doctest: +SKIP
-              x  y energy num_oc. chain_.
-           0 -1 +1   -1.0       6     0.0
-           1 +1 -1   -1.0       4     0.0
-           ['SPIN', 2 rows, 10 samples, 2 variables]
+            This example uses :class:`.VirtualGraphComposite` to instantiate a 
```

Comment:
Is this the same example as the one above in the class docs? If yes, why duplicating it 50 lines apart?

Reply:
Removed duplicate